### ISSUE TITLE
Hotfix 12 issues.

### ISSUE THREAD

**Issue by honeydog7** [Norbyte/bg3se#193]
Patch 5 didn't break modded game. **Hotfix 12 did**. Was working smoothly, as mod-updates came in, before the hotfix. 

Relatively certain it's a Script Extender issue because Transmog Enhanced, Rich Traders, and any other SE-dependent mod (all were working post Patch 5, pre-Hotfix) are now broken. Particularly, Appearance Edit is causing the most dire issues:

1) Cannot mirror or re-sculpted Tav or partymembers. Spell is no longer in spellbook. 
2) NPC partymembers cannot enter mirror in camp.
3) Previously re-sculpted NPC partymembers (before hotfix 12) now appear as the generic race/subrace/body-type for character creation. **This causes severe issues when the body type was previously changed.** makes game unplayable and cannot uninstall mod.

Attempted the following:

1) Re-downloaded and reinstalled SE, Improved UI, AEE, and reprioritized load order.
2) Verified game files.
3) All three time-stamps of SE v.11 both exporting default values, and not.

**Comment by Kvalyr** [comment]
These issues all look mod-specific and aren't anything to do with Script Extender itself. Some mods need to be updated to account for new versions of the game, or of the script extender. 
Mod authors will get around to them when they have some free time and energy to do it - but this isn't the right place to report issues with specific mods.

**Comment by honeydog7** [comment]
> These issues all look mod-specific and aren't anything to do with Script Extender itself. Some mods need to be updated to account for new versions of the game, or of the script extender. Mod authors will get around to them when they have some free time and energy to do it - but this isn't the right place to report issues with specific mods.

You probably just gave a cursory glance at the issue. SE's lack of update to the hotfix is getting people locked out of games. A lot of game code has been changed in the last 36 hours.

I used Appearance Edit to illustrate the dependency issue-- these mod authors are reflexive enough to have already updated to Patch 5. SE has not yet adjusted to the newest hotfixed version of the game (to the point that there's a pop-up when one launches from BG3MM that let's you know). If I'm doing something wrong in using this free tool, please advise... but I'm relatively certain that the hotfix has altered code in the game where v.11 as of Nov. 30, is obsolete without a few tweaks. 

I've seen others jump on people in this forum, either individualizing the issue: "user-error" or defending the tool itself and passing the buck to individual mod developers. It is a brilliant tool, I agree-- but the point is to raise issues and make it better so we CAN use mods without quitting our day jobs.

**Comment by FallenStar08** [comment]
QUICK SOMEONE CALL NORBYTE A LOT OF CODE HAS CHANGED IN THE LAST 36 HOURS, I HAVE MY SOURCES

**Comment by Kvalyr** [comment]
@FallenStar08 Maybe cool it with the trolling a bit - it's not constructive or helpful.

@honeydog7 You assert that "a lot of code has changed" - What's your evidence for that?
I'm not a dev of Script Extender itself and I don't use the mods you've given as examples, but I am actually one of the mod authors that I referred to in my original reply; and I can say for sure that some of the mods you've cited last received an update **before** Patch5 came out; not since.

The SE mods that I do use (and develop as a mod author) are working fine with BG3 Patch5/Hotfix12. I haven't seen anything to suggest that Script Extender itself is faulty; and I stand by my assertion that some mods may need to be updated to account for changes before you can expect them to work correctly.

**Comment by yawor666** [comment]
so what is the consensus here? does SE work with patch 5, hotfix 12 (game ver 4.1.1.4079877 GOG)?
i'm waiting with the update because it's over 100GB on GOG for some reason and i don't fancy wasting my time only to go back to patch 4 / hotfix 3 :)
i'am aware, that probably not all the mods i use got updated already, asking strictly about SE :)

**Comment by honeydog7** [comment]
> i'am aware, that probably not all the mods i use got updated already, asking strictly about SE :)

On my end, I figured out the issue with Hotfix 12 this morning. User error (maybe?) with BG3MM? I've been jerry-rigging the game, in order to get the mods to work between patch 3 and 4, using different versions of SE within the 'Developer' channel. Never thought to use the 'Release' channel. So the latest version of SE I've used was 30Nov... 'Release' is 01Dec and it seems to work with the mods as I add them into a new game. It was right in front of me. The errors I do see in the console, from what I can tell (my career had me doing code two decades ago and it is a perishable skill), looks like stuff that needs to be sorted out by specific mod developers when they get the time-- this is an avocation after all. 

Hotfix 12 definitely changed a lot of things, otherwise 30Nov SE would work fine. It worked smoothly prior to the hotfix.

Luckily, 01Dec 'Release' SE  works. I just wish version selection was a little more explicit in the Developer channel of BG3MM, and that it didn't just say "Latest", but actually plugged in a date. My dumbass kept assuming 30Nov was the the be-all, end-all that everyone kept saying should work. My bad.  

I play a game that's probably got a lot of mods (I don't know what others do) and it now seems to work with Patch 5/Hotfix 12. This includes stuff like Appearance Edit, Rich Traders, Transmog, et cetera...
